src/components/event_preview.js

```diff
@@ -17,6 +17,30 @@ function makeFakeAuxDataForEventOptions(event) {
   };
 }
 
+function makeFakeAuxDataForSulis(event) {
+
+  const fakeSulisAuxData = {
+
+    data_source : "vehicleRealtimeFramegrabberData",
+    data_array: [
+      {
+        data_name: "camera_name",
+        data_value: "Sulis Camera",
+        data_uom: "",
+      },
+      {
+        data_name: "filename",
+        data_value: event.event_options[0].event_option_value,
+        data_uom: "",
+      }
+    ]
+
+  }
```

Comment:
Thanks for splitting this into a separate function, that looks good.

This should have a `;` at the end of the line. In JavaScript they are optional, but try to be consistent with the surrounding code.

You don't have to go back to fix it since it's not affecting anything, and you may have bigger fish to fry right now, but if you ever want to do a quick cosmetic change like this:

If it's a problem with the very last commit, you can just make the change, `git add` the file, and then do `git commit --amend`.

If it's a problem with an earlier commit, often what I'll do is:

1. Make the fix in a separate commit. The commit message can be any dummy string that's easy to identify, since it's only temporary.
2. Use `git rebase -i main`. When the menu opens up, find your fix commit. Reorder the list so that it comes immediately after the commit it is fixing. Then change it to a `fixup`. This will flatten the two changes, keeping the original commit message and dropping the dummy one.